templates/docs/examples/patterns/adv-grid/default.html

```diff
@@ -0,0 +1,61 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Advanced Grid / Default{% endblock %}
+
+{% block standalone_css %}patterns_adv-grid{% endblock %}
+
+{% block content %}
+
+<div class="adv-grid-demo">
+  <div class="p-adv-grid">
+    <div class="p-adv-grid__title">
+      <p>MORE FROM CANONICAL</p>
+    </div>
+    <div class='p-adv-grid__row'>
+      <hr class='p-adv-grid__row-divider--below-header' />
+      <div class="p-adv-grid__col">
+        <strong class="p-adv-grid__col-header">
+          Some Title
+        </strong>
+        <p class="p-adv-grid__col-content">Use the Ubuntu terminal and run Linux applications on Windows. Use the Ubuntu
+          terminal
+          and run Linux applications on Windows.</p>
+        <footer class="p-adv-grid__col-footer">
+          <hr />
```

Comment:
Our default for horizontal lines is rule component, so `<hr class="p-rule" />` this makes sure there is no additional unnecessary margin.